components/gitpod-db/src/typeorm/blocked-repository-db-impl.ts

```diff
@@ -23,6 +23,37 @@ export class TypeORMBlockedRepositoryDBImpl implements BlockedRepositoryDB {
         return (await this.getEntityManager()).getRepository<DBBlockedRepository>(DBBlockedRepository);
     }
 
+    public async findAllBlockedRepositories(
+        offset: number,
+        limit: number,
+        orderBy: keyof BlockedRepository,
+        orderDir: "DESC" | "ASC",
+        searchTerm?: string,
+        minCreationDate?: Date,
+        maxCreationDate?: Date,
+    ): Promise<{ total: number; rows: BlockedRepository[] }> {
+        const blockedRepositoryRepo = await this.getBlockedRepositoryRepo();
+
+        const qBuilder = blockedRepositoryRepo.createQueryBuilder("br");
+        if (searchTerm) {
+            qBuilder.andWhere(`br.urlRegexp LIKE :searchTerm`, { searchTerm: "%" + searchTerm + "%" });
```

Comment:
Note that this won't work: `andWhere` strictly requires a prior `where` this is a typeorm oddity... :see_no_evil: 
Usually it can be solved by having something general as first clause, maybe `.where("br.deleted = FALSE")`.

Reply:
> This does seem to work though; tests using the searchTerm pass and it behaves as expected when tested manually too.

Uh, that's nice! Might have been introduced with the typeorm version bump we did last October - thx for sharing!